README.md

```diff
@@ -0,0 +1,5 @@
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.

```

Comment:
create a more descriptive README.md

Reply:
It is nice that you are advising the author to do the correct thing, but the tone used is an **_authoritative_** one. This is the same case with other inline comments you gave in this project. The authoritative tone used in communicating changes required can **_demoralize_** the morale of the author and make him defensive instead of **_open_** to corrections.

> Kindly remember that as a rule of thumb, one major reason for performing code reviews is to make communication within a team faultless. Using authoritative tones when communicating ideas would defeat this purpose.